### ISSUE TITLE
Structure pattern accessibility: add a label to the search input

### ISSUE THREAD

**Issue by erral** [plone/mockup#952]
In an accessibility, the reviewer says that the search box in the structure pattern lacks a label to improve its accessiblity

@ionlizarazu
